app/views/idv/verify_info/show.html.erb

```diff
@@ -54,7 +54,9 @@ locals:
       </div>
       <div>
         <dt class="display-inline"> <%= t('idv.form.dob') %>: </dt>
-        <dd class="display-inline margin-left-0"> <%= @pii[:dob] %> </dd>
+        <dd class="display-inline margin-left-0">
+          <%= DateParser.parse_legacy(@pii[:dob]).to_formatted_s(:long) %>
```

Comment:
Would the format of the date ever be in the "legacy" American format that we'd need to use `parse_legacy`?

```suggestion
          <%= Date.parse(@pii[:dob]).to_formatted_s(:long) %>
```

Reply:
Question: Does `to_formatted_s` work the way we'd expect for French and Spanish? I know for some other time formatting we've used locale strings, unsure if we should be doing the same here.

https://github.com/18F/identity-idp/blob/6183c2e7d7594b47d57293b833cc748c90d762bf/config/locales/time/en.yml#L31-L37

Reply:
I see the implementation might have been inherited by `_pii_review.html.erb`, but now I'm wondering if that existing view might be buggy for non-English locales? 🤔 

At least in REPL:

```
$ rails c
> I18n.locale = :es
=> :es
> Date.parse('2022-02-01').to_formatted_s(:long)
=> "February 01, 2022"
> I18n.l(Date.parse('2022-02-01'), format: I18n.t('time.formats.event_date'))
=> "1 de febrero 2022"
```

Reply:
Good catch! Per Slack discussion I'll use `I18n.l(Date.parse...` here because this isn't long-stored db data that might be in a legacy format, and `_pii_review.html.erb` issues will be addressed with a separate ticket.